docs/modules/druid/pages/usage-guide/security.adoc

```diff
@@ -0,0 +1,122 @@
+= Security
+
+The Druid cluster can be secured and protected in multiple ways.
+
+== Encryption
+
+TLS encryption is supported for internal cluster communication (e.g. between Broker and Coordinator) as well as for external communication (e.g. between the Browser and the Router Web UI).
+
+[source,yaml]
+----
+spec:
+  clusterConfig:
+    tls:
+      serverAndInternalSecretClass: tls # <1>
+----
+<1> Name of the `SecretClass` that is used to encrypt internal and external communication.
+
+IMPORTANT: A Stackable Druid cluster is always encrypted per default. In order to disable this default behavior you can set `spec.clusterConfig.tls.serverAndInternalSecretClass: null`.
```

Comment:
```suggestion
IMPORTANT: A Stackable Druid cluster is always encrypted per default i.e. `spec.clusterConfig.tls.serverAndInternalSecretClass: tls` in the above example does not need to be specified as it will be applied by default: in order to disable this default behavior you can set `spec.clusterConfig.tls.serverAndInternalSecretClass: null`.
```